x-pack/plugins/security_solution/public/resolver/view/panels/panel_content_utilities.tsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { EuiBreadcrumbs, Breadcrumb } from '@elastic/eui';
+import styled from 'styled-components';
+import React, { memo } from 'react';
+import { useResolverTheme } from '../assets';
+
+/**
+ * The two query parameters we read/write on to control which view the table presents:
+ */
+export interface CrumbInfo {
+  readonly crumbId: string;
+  readonly crumbEvent: string;
+}
+
+const ThemedBreadcrumbs = styled(EuiBreadcrumbs)<{ background: string; text: string }>`
+  &.euiBreadcrumbs.euiBreadcrumbs--responsive {
+    background-color: ${(props) => props.background};
+    color: ${(props) => props.text};
+    padding: 1em;
+  }
+`;
+
+/**
+ * Breadcrumb menu with adjustments per direction from UX team
+ */
+export const StyledBreadcrumbs = memo(function StyledBreadcrumbs({
+  breadcrumbs,
+  truncate,
+}: {
+  breadcrumbs: Breadcrumb[];
+  truncate?: boolean;
+}) {
+  const {
+    colorMap: { resolverEdge, resolverEdgeText },
+  } = useResolverTheme();
+  return (
+    <ThemedBreadcrumbs
+      background={resolverEdge}
+      text={resolverEdgeText}
+      breadcrumbs={breadcrumbs}
+      truncate={truncate}
+    />
+  );
+});
+
+/**
+ * Long formatter (to second) for DateTime
+ */
+export const formatter = new Intl.DateTimeFormat(i18n.getLocale(), {
+  year: 'numeric',
+  month: '2-digit',
+  day: '2-digit',
+  hour: '2-digit',
+  minute: '2-digit',
+  second: '2-digit',
+});
+
+/**
+ * @param {ConstructorParameters<typeof Date>[0]} timestamp To be passed through Date->Intl.DateTimeFormat
+ * @returns {string} A nicely formatted string for a date
+ */
+export function formatDate(timestamp: ConstructorParameters<typeof Date>[0]) {

```

Comment:
Why not use `FormattedDate`?

Reply:
I was taking the work you did in the panel before here and just putting it inside of a function. I guess it could return a `<FormattedDate>`. I'll see how that plays out.